zeebe/util/src/main/java/io/camunda/zeebe/util/TagUtil.java

```diff
@@ -9,13 +9,20 @@
 
 import java.util.regex.Pattern;
 
+/**
+ * Utility class for validating tags used in Zeebe. Tags are used to categorize and label resources
+ * within the system.
+ *
+ * <p>NOTE: Whenever you adjust the tag format or max number, also consider updating the REST API
+ * documentation and the duplicated Camunda client validation (see TagUtil.java)
+ */
 public class TagUtil {
 
   public static final int MAX_NUMBER_OF_TAGS = 10;
```

Comment:
💭 Should we make this configurable ? Why 10 has been chosen ?

Reply:
Maybe we want to in the future, but not in MVP state - I just choose a sensible number and got confirmation from Aleksander. We also don't want too many tags 😄 so when we allow to configure it then user might escalate 🦀